src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ControlDesigner.ChildWindowTarget.cs

```diff
@@ -61,7 +61,7 @@ public void OnMessage(ref Message m)
                 // If the designer disposed us, then we should follow suit.
                 if (_designer.DesignerTarget is null)
                 {
-                    designerTarget.Dispose();
+                    designerTarget!.Dispose();
```

Comment:
I wouldn't let this fail.
```suggestion
                    designerTarget?.Dispose();
```